pkg/commands/kustfile/kustomizationfile_test.go

```diff
@@ -176,10 +175,11 @@ func TestPreserveCommentsWithAdjust(t *testing.T) {
 
     
 
-# shem qing some comments
+# Some comments
 # This is some comment we should preserve
 # don't delete it
-resources:
+RESOURCES:
```

Comment:
Why do you use all capital letters here?

Reply:
its just a YAML test that was there before.
the conversion lowercases the field names.

Reply:
Sounds good.